### PR TITLE
feat: Gravewood Grove Zone

### PR DESCRIPTION
**Pull request by Mindbomber2**

### CODE REVIEW

File: src/main/java/spireMapOverhaul/zones/gravewoodGrove/TransformOption.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import spireMapOverhaul.zones.gremlincamp.GremlinCamp;
+import spireMapOverhaul.zones.gremlincamp.ScavengeEffect;
+
+import static spireMapOverhaul.SpireAnniversary6Mod.makeID;
+import static spireMapOverhaul.SpireAnniversary6Mod.makeUIPath;
+
+public class TransformOption extends AbstractCampfireOption {
+    private static final UIStrings uiStrings = CardCrawlGame.languagePack.getUIString(makeID("TransformOption"));
+    public static final String[] TEXT = uiStrings.TEXT;
+    protected int minGold, maxGold;
```
Comment (erasels) [inline src/main/java/spireMapOverhaul/zones/gravewoodGrove/TransformOption.java:19]: Seems useless here, might as well remove it

File: src/main/java/spireMapOverhaul/zones/gravewoodGrove/powers/DeadBranchPower.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    private int cardsExhaustedThisTurn = 0;
+
+
+    public DeadBranchPower(AbstractCreature owner) {
+        super(POWER_ID, NAME, AbstractPower.PowerType.DEBUFF, false, owner, 0);
+    }
+
+    public void onExhaust(AbstractCard card) {
+        if (!AbstractDungeon.getMonsters().areMonstersBasicallyDead()) {
+            flash();
```
Comment (erasels) [inline src/main/java/spireMapOverhaul/zones/gravewoodGrove/powers/DeadBranchPower.java:31]: Maybe use flashWithoutSound() instead

File: src/main/java/spireMapOverhaul/zones/gravewoodGrove/powers/DeadBranchPower.java [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+            addToBot(new RelicAboveCreatureAction(AbstractDungeon.player, RelicLibrary.getRelic("Dead Branch")));
```
Comment (erasels) [inline src/main/java/spireMapOverhaul/zones/gravewoodGrove/powers/DeadBranchPower.java:32]: Don't use string literals for relic IDs

File: src/main/java/spireMapOverhaul/zones/gravewoodGrove/powers/DeadBranchPower.java [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+            addToBot(new MakeTempCardInHandAction(AbstractDungeon.returnTrulyRandomCardInCombat().makeCopy(), false));
```
Comment (erasels) [inline src/main/java/spireMapOverhaul/zones/gravewoodGrove/powers/DeadBranchPower.java:33]: The action itself already makes a copy of the card, but it's fine

**Review by erasels — CHANGES_REQUESTED** [review]

File: src/main/java/spireMapOverhaul/zones/gravewoodGrove/powers/DeadBranchPower.java (hunk 4 above)
Comment (Mindbomber2) [reply src/main/java/spireMapOverhaul/zones/gravewoodGrove/powers/DeadBranchPower.java:34]: keeping it in as discussed

**Pull request merged**